docs/UserGuide.md

```diff
@@ -356,47 +350,104 @@ A lesson **must** have the following fields: a start date, a time range, a lesso
 A lesson can be categorised into 2 types: 
 1. A **weekly** recurring lesson
 2. A one-off makeup lesson.
-
-The essential fields for a lesson are:
-* Start date
-* Time range
-* Subject
-* Rate
-
-An optional field for both types of lesson is:
-* Homework
-* Outstanding Fees (refer to more details in this [section](#managing-lesson-fees))
-
-Optional fields for a **recurring** lesson is:
-* End date
-* Cancelled dates
-
-TAB has features to help to prevent scheduling mistakes.
-For example, we help you avoid mistakes such as scheduling two lessons at the same slots by ensuring that lessons never overlap.
-Additionally, we help to avoid accidentally scheduling lessons outside working hours (before 0800 and after 2200 hours).
-
-In the future, we intend to add some nice-to-have features that will allow you to customise your working hours,
-as well as specify minimum/maximum lesson durations so that accidents like mistyping a 10-minute lesson won't happen.
-Additionally, we will allow you to toggle these checks on and off, just in case you don't want them.
-Do look forward to these features!
-
-<div markdown="block" class="alert alert-info">
-**:information_source: Note:**<br>
-* The lesson's rate refers to the fee of the lesson per hour.
-This rate will be used in the calculation of fees due after each lesson.
-
-* A lesson can be identified by the index number shown in the lesson list of the student.
-</div>
-
-<br>
+   
+In the future, you can also customise your own recurrence rule for your recurring lessons.
+
+A lesson can be identified by the index number shown in the lesson list of the student.
+
+<table id="student-param-table">
+    <thead>
+        <tr>
+            <th style="text-align:center; padding: 10px">Category</th>
+            <th style="text-align:center">Field</th>
+            <th style="text-align:center">Prefix</th>
+            <th style="text-align:center">Constraint(s)</th>
+            <th style="text-align:center">Example(s)</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td rowspan=4>Essential</td>
+            <td>Start Date</td>
+            <td><code>date/</code></td>
+            <td>
+              <ul> 
+                <li>Must be present.</li>
+                <li>Case-insensitive.</li>
```

Comment:
remove mention of this is `ladd`

Reply:
this one -> `The date is case-insensitive. i.e. 12 jaN 2022 is equivalent to 12 JAN 2022.`